coap-gateway/config.yaml

```diff
@@ -6,6 +6,7 @@ apis:
     address: "0.0.0.0:5684"
     externalAddress: ""
     maxMessageSize: 262144
+    cacheExpiration: 10m
```

Comment:
It's not descriptive enough. ownerCachceExpiration? Should it be really limited in the case of CoAP Gateway which should keep the cache open as long as the device is connected? There shouldn't be imho any time-based expiration.

Reply:
It's nice when you connect `1000 devices` at the same time so it will use data from the updated cache.

Reply:
Decrease to 1m

Reply:
Updated